build.sbt

```diff
@@ -14,6 +14,7 @@ val akkaV = "2.3.12"
 
 libraryDependencies ++= Seq(
   "org.scala-lang" % "scala-reflect" % "2.11.7",
+  "com.gettyimages" %% "spray-swagger" % "0.5.1",
```

Comment:
We no longer use `spray-swagger`. This is leftover code cruft. Will point out the other parts below.